src/osha/oira/client/browser/client.py

```diff
@@ -295,15 +321,11 @@ def get_addresses_for_groups(self, group_paths):
                 continue
             # Special handling for language specific mailing lists, e.g. "be-fr"
             country, lang = group_id.split("-")
-            country_subscribers = (
-                Session.query(Account.loginname)
-                .filter(Account.id == NewsletterSubscription.account_id)
-                .filter(NewsletterSubscription.zodb_path == (country))
-                .filter(Account.id == NewsletterSetting.account_id)
-                .filter(NewsletterSetting.value == f"language:{lang}")
-                .group_by(Account.loginname)
-            )
-            subscribers.extend([s.loginname for s in country_subscribers])
+            if lang == "managers":
+                # Not actually a language, but the special country managers test list
+                subscribers.extend(self.get_addresses_of_country_managers(country))
+            else:
+                subscribers.extend(self.get_addresses_per_language(country, lang))
```

Comment:
I have the feeling that instead of using a list a set would be better.
Did you consider that?

Reply:
I haven't considered it, no. Thinking about it now I don't see a big benefit. There's not much chance of getting duplicates - users usually don't subscribe to mailing lists of multiple countries/languages, and I don't think newsletters are sent to multiple countries either, let alone in multiple languages.

Do you want me to switch anyway?

Reply:
Ah what the heck, I'll switch. It probably makes sense.

Reply:
Pushed. Thanks for the suggestion.